distributed/protocol/pickle.py

```diff
@@ -33,15 +33,15 @@ def _always_use_pickle_for(x):
         return False
 
 
-def dumps(x, *, buffer_callback=None):
+def dumps(x, *, buffer_callback=None, protocol=HIGHEST_PROTOCOL):
```

Comment:
Is this suppose to be `None`? Asking as below we have `protocol or HIGHEST_PROTOCOL`, which seems unneeded if this is `HIGHEST_PROTOCOL`.

Reply:
Either one is fine with me.  We sometimes pass in `protocol=None`, hence the `or HIGHEST_PROTOCOL` bit, but I figured having the default be informative wouldn't hurt in either case.

Reply:
Yeah no strong feelings from me either. Just noting it's a little odd to have `HIGHEST_PROTOCOL or HIGHEST_PROTOCOL` potentially ;)